src/Avalonia.Controls/WrapPanel.cs

```diff
@@ -67,12 +79,30 @@ public class WrapPanel : Panel, INavigableContainer
         /// </summary>
         static WrapPanel()
         {
-            AffectsMeasure<WrapPanel>(OrientationProperty, ItemWidthProperty, ItemHeightProperty);
+            AffectsMeasure<WrapPanel>( ItemSpacingProperty, LineSpacingProperty,OrientationProperty, ItemWidthProperty, ItemHeightProperty);
```

Comment:
Nit: remove the extra space before the first parameter, add one after the second comma.